docs/content/preview/admin/yb-admin.md

```diff
@@ -41,7 +41,7 @@ yb-admin \
 To display the online help, run `yb-admin --help` from the YugabyteDB home directory.
 
 ```sh
-$ ./bin/yb-admin --help
+bin/yb-admin --help
```

Comment:
I thought the user would need to copy and paste `./bin/yb-admin --help`

Reply:
Yes, Thanks for the clarification again @lizayugabyte . Will make the changes.